backend/kor/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Use an official Golang image as the build environment
+FROM golang:1.22.2-alpine AS builder
```

Comment:
Take a look at the kor dockerfile https://github.com/yonahd/kor/blob/main/Dockerfile